### PR TITLE
Update variables documentation

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

Prior to work on variables, I didn't fully understand this document. I've changed it so it makes a lot more sense.

### CODE REVIEW

**Comment by alazymeme** [comment]
Any chance someone could assist with the `add` and `slice` filters? I couldn't figure out what add does, slice I'm not 100% how to word to make it "easier" to understand.

**Comment by RAnders00** [comment]
add() interprets the input as a number and adds some other number to it. E.g. `$(datetime:Europe/Amsterdam|strftime(%W)|add(1)` gets next week's week-of-the-year number.

slice() is documented here: https://github.com/pajbot/pajbot/pull/812

File: docs/variables.md [hunk 1]
```diff
@@ -17,25 +17,22 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
-"join": \_filter_join,\
-"number_format": \_filter_number_format,\
-"add": \_filter_add,\
-"or_else": \_filter_or_else,\
-"or_broadcaster": self.\_filter_or_broadcaster,\
-"or_streamer": self.\_filter_or_broadcaster,\
-"slice": \_filter_slice,
+`strftime` - allows you to format raw time strings - Help: https://strftime.org\
+`lower` - converts the variable all to lowercase\
+`upper` - converts the variable all to uppercase\
```
Comment (pajlada) [inline docs/variables.md:22]: rephrase to real english please

File: docs/variables.md [hunk 2]
```diff
@@ -17,25 +17,22 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
+`swapcase` - swaps all upper case to lower case and vice versa\
+`time_since_minutes` - outputs the time since a certain input time in minutes\
+`time_since` - outputs the time since a certain input time\
+`time_since_dt` - outputs the date since a certain input date\
+`urlencode` - replace spaces with a `+`\
+`join` - join characters together using `,`\
+`number_format` - format numbers using `,`\
+`add` - ????\
+`or_else` - if nothing, return what is specified\
+`or_broadcaster`/`or_streamer` - if nothing, return the streamer's name\
```
Comment (pajlada) [inline docs/variables.md:34]:
```suggestion
`or_broadcaster`/`or_streamer` - if the previous filter failed, return the streamer's name\
```

File: docs/variables.md [hunk 3]
```diff
@@ -17,25 +17,22 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
+`capitalize` - converts the first letter of the first word to upper case\
+`swapcase` - swaps all upper case to lower case and vice versa\
+`time_since_minutes` - outputs the time since a certain input time in minutes\
+`time_since` - outputs the time since a certain input time\
+`time_since_dt` - outputs the date since a certain input date\
+`urlencode` - replace spaces with a `+`\
+`join` - join characters together using `,`\
+`number_format` - format numbers using `,`\
+`add` - ????\
+`or_else` - if nothing, return what is specified\
```
Comment (pajlada) [inline docs/variables.md:33]:
```suggestion
`or_else` - if the previous filter failed, return what is specified\
```

File: docs/variables.md [hunk 4]
```diff
@@ -17,25 +17,22 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
+`strftime` - allows you to format raw time strings - Help: https://strftime.org\
+`lower` - converts the variable all to lowercase\
+`upper` - converts the variable all to uppercase\
+`title` - converts the first letter in each word to upper case\
+`capitalize` - converts the first letter of the first word to upper case\
+`swapcase` - swaps all upper case to lower case and vice versa\
+`time_since_minutes` - outputs the time since a certain input time in minutes\
+`time_since` - outputs the time since a certain input time\
+`time_since_dt` - outputs the date since a certain input date\
+`urlencode` - replace spaces with a `+`\
```
Comment (pajlada) [inline docs/variables.md:29]:
```suggestion
`urlencode` - URL encodes the value. see https://en.wikipedia.org/wiki/Percent-encoding\
```

File: docs/variables.md [hunk 5]
```diff
@@ -17,25 +17,22 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
+`join` - join characters together using `,`\
+`number_format` - format numbers using `,`\
+`add` - ????\
+`or_else` - if nothing, return what is specified\
+`or_broadcaster`/`or_streamer` - if nothing, return the streamer's name\
+`slice` - ????
```
Comment (pajlada) [inline docs/variables.md:35]:
```suggestion
`slice` - slice the string like a python3 string - see https://www.digitalocean.com/community/tutorials/how-to-index-and-slice-strings-in-python-3 (NOTE: We don't support what they call 'stride')
```

**Review by pajlada — CHANGES_REQUESTED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**